<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileShareWidget</class>
 <widget class="QWidget" name="FileShareWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Local File Share &amp; Sync Server</string> </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Start the server to enable local Wi-Fi file sharing and clipboard sync (PC-to-device). Other devices connect using their web browser. No mobile app needed for basic functionality.</string> </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="toggleServerButton">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start Server</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="serverInfoGroup">
     <property name="title">
      <string>Connection Info</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="urlLabel">
          <property name="text">
           <string>Status: Server is OFF</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;b&gt;For File Transfer &amp; Basic Clipboard View:&lt;/b&gt;&lt;br&gt;On another device (phone, tablet, laptop) on the same Wi-Fi, open its web browser and go to the URL above, or scan the QR code.&lt;br&gt;&lt;br&gt;This web page allows file upload/download and shows clipboard text sent from this PC.</string> </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="qrCodeLabel">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #eee;</string>
        </property>
        <property name="text">
         <string>QR Code</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Notifications:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="notificationList"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
